examples/single-page-app/myhub/api.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+import json
+import logging
+import os
+import pathlib
+import secrets
+import urllib.parse
+
+import yaml
+
+from aiohttp import web
+
+from shared import Data, debug_request, TokenStorage
+
+MYHUB_URL = os.environ.get("MYHUB_URL") or "http://localhost:7000"
+
+token_storage = TokenStorage(pathlib.Path(os.environ["TOKEN_STORAGE_PATH"]))
```

Comment:
I'm probably dense, but can you explain who is writing this in the prod use case?

Reply:
myhub is for demo purposes only

it has a "production" version - ie tls/gzip/static assets - but its very much not for production - as the comments make clear (at least in the adjacent file) - i can add here also

Reply:
ive tried to make the health warnings here a bit more obvious/prominent